1.9/api/logs.yaml

```diff
@@ -0,0 +1,216 @@
+swagger: '2.0'
+
+info:
+  version: "0.0.1"
+  title: DC/OS Log API
+  description: >
+    The first version of the DC/OS Log API.
+    Note that the endpoints in this spec are all *service-specific*. dcos-log
+    is designed to be put behind Admin Router, which provides security for these
+    endpoints. Therefore, on a DC/OS cluster, the final URL to query will resemble:
+        http://<host>/system/v1/agent/<agent-id>/logs/v1/<endpoint>
+    For more information, please consult the project documentation located at
```

Comment:
This file is managed by the day2ops team, copied from the source repo: https://github.com/dcos/dcos-log/blob/master/swagger.yml